lib/common/api/deprecate.ts

```diff
@@ -53,6 +55,18 @@ const deprecate: ElectronInternal.DeprecationUtil = {
     })
   },
 
+  // deprecate a getter/setter function in favor of a property
+  fnToProperty: (fn, propName) => {
```

Comment:
Because we're going to be doing these in pairs, would be make sense for `fn` to either be an array of functions or just take in two functions, a getter and a setter